gef.py

```diff
@@ -6346,6 +6352,77 @@ def empty_extra_messages(self, event):
         return
 
 
+@register_command
+class MemoryCommand(GenericCommand):
+    """Add or remove address ranges to the memory view."""
+
+    _cmdline_ = "memory"
+    _syntax_  = """{cmd:s} watch ADDRESS [SIZE] [(qword|dword|word|byte)]
+{cmd:s} unwatch ADDRESS
+{cmd:s} clear
+{cmd:s} list""".format(cmd=_cmdline_)
+    _example_ = "{:s} watch 0x603000 0x100 byte".format(_cmdline_)
+
+    def __init__(self):
+        super(MemoryCommand, self).__init__(prefix=True)
+        global watches
+        watches = {}
+        return
+
+    @only_if_gdb_running
+    def do_invoke(self, argv):
+        if len(argv) < 1:
+            self.usage()
+            return
+        cmd, opt = argv[0], argv[1:]
```

Comment:
Yeah i tried, it wasn't working and I couldn't figure out why. I copied from a sample (heap bins, etc), but it wasn't working.